faq.html

```diff
@@ -4,4 +4,16 @@
 ---
 
 <h3>Frequently Asked Questions</h3>
-todo
\ No newline at end of file
+1. Why the name "in-toto"?
+in-toto is Latin for “as a whole". It is the first framework that holistically enforces the integrity of a software supply chain by gathering cryptographically verifiable information about the chain itself, from development to installation of the software.
```

Comment:
I like this, but I wonder if we want to keep these the answers a little shorter. What do you think of:
`in-toto is Latin for "as a whole". We chose that name because it's a system to protect the whole software supply chain`

Reply:
Sounds good. I have made one small change to your suggestions in the new PR.

Reply:
Here's what I have added:
"in-toto is Latin for “as a whole". We chose the name because our objective with in-toto is to build a system to protect the whole software supply chain."